content-api/content-actors/src/main/scala/org/sunbird/content/util/AcceptFlagManager.scala

```diff
@@ -0,0 +1,93 @@
+package org.sunbird.content.util
+
+import java.util
+
+import org.apache.commons.lang.StringUtils
+import org.sunbird.cache.impl.RedisCache
+import org.sunbird.common.dto.{Request, Response, ResponseHandler}
+import org.sunbird.common.exception.ResponseCode
+import org.sunbird.graph.OntologyEngineContext
+import org.sunbird.graph.nodes.DataNode
+import org.sunbird.graph.dac.model.Node
+import org.sunbird.graph.external.ExternalPropsManager
+import org.sunbird.graph.utils.ScalaJsonUtils
+import org.sunbird.managers.HierarchyManager
+
+import scala.concurrent.{ExecutionContext, Future}
+
+object AcceptFlagManager {
+
+  def acceptFlag(request: Request)(implicit ec: ExecutionContext, oec: OntologyEngineContext): Future[Response] = {
+    DataNode.read(request).map(node => {
+      if (!StringUtils.equals(ContentConstants.CONTENT_OBJECT_TYPE, node.getObjectType) ||
```

Comment:
```suggestion
      if (!StringUtils.equals(ContentConstants.CONTENT_OBJECT_TYPE, node.getObjectType) ||
```
This validation is not required. It will be handled at Ontology engine layer.